main.css

```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #efefef;
+}
+
+.container {
+  position: absolute;
```

Comment:
Check out the flex spec for centering content. It's super easy when you get the hang of it: https://www.kirupa.com/html5/centering_vertically_horizontally.htm